### PR TITLE
Update publisher landing page with benefits

### PR DESCRIPTION
**Pull request by ericholscher**
This focuses on our benefit to publishers.
Less about us, more about them.

The primary things I focused on:

* Transparency (understand what's going on)
* Privacy (no banners or regulation issues)
* Control (good design, no pressure)

## Preview

127 0 0 1_8000_publishers_

### CODE REVIEW

File: content/pages/publishers.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Title: Earn Money with Ethical Advertising
+Title: Publisher: Earn money on your developer content
```
Comment (davidfischer) [inline content/pages/publishers.md:1]: Having publisher in the title is probably good on a "publisher" page. Should it be "developer project"? I don't feel strongly.

File: ethicalads-theme/templates/ea/includes/publisher-testimonials.html [hunk 2]
```diff
@@ -3,9 +3,9 @@
 
 <section class="container marketing py-10 text-center">
 
-  <h2 class="font-weight-bold mb-8">Some of our publishers</h2>
+  <h2 class="font-weight-bold mb-8">You're in great company</h2>
 
-  <p class="font-size-lg text-gray-800 w-75 mx-auto mb-7 mb-md-9">Learn about how we help developers monetize their project and keep their development efforts sustainable.</p>
+  <p class="font-size-lg text-gray-800 w-75 mx-auto mb-7 mb-md-9">These developer-focused sites have found great success with EthicalAds, you could too!</p>
```
Comment (davidfischer) [inline ethicalads-theme/templates/ea/includes/publisher-testimonials.html:8]:
```suggestion
  <p class="font-size-lg text-gray-800 w-75 mx-auto mb-7 mb-md-9">These developer-focused sites have found great success with EthicalAds. You will too!</p>
```

File: ethicalads-theme/templates/ea/includes/publisher-testimonials.html [hunk 3]
```diff
@@ -68,7 +68,7 @@ <h2 class="font-weight-bold mb-8">Some of our publishers</h2>
           </div>
 
           <!-- Text -->
-          <p class="text-gray-700 mb-5 small">Flask is one of the most popular Python web development frameworks. They use our EthicalAds to get funding for their community to maintain their software.</p>
+          <p class="text-gray-700 mb-5 small">Flask is one of the most popular Python web development frameworks. They use  EthicalAds to fund the ongoing maintaince of their open source project.</p>
```
Comment (davidfischer) [inline ethicalads-theme/templates/ea/includes/publisher-testimonials.html:71]:
```suggestion
          <p class="text-gray-700 mb-5 small">Flask is one of the most popular Python web development frameworks. They use EthicalAds to fund the ongoing maintaince of their open source project.</p>
```

File: ethicalads-theme/templates/ea/publishers.html [hunk 4]
```diff
@@ -100,8 +100,8 @@ <h3>Publisher Friendly</h3>
     <div class="row justify-content-center">
 
       <div class="col-md-6 mb-3">
-        <h2 class="font-weight-bold mb-8">One tasteful ad</h2>
-        <p class="font-size-lg text-gray-800">We only show one ad per page, because developers don't want ads all over their sites. Advertising is a trade-off that we choose, to make our projects more sustainable. We don't compromise on privacy though.</p>
+        <h2 class="font-weight-bold mb-8">One ad that compliments your content</h2>
+        <p class="font-size-lg text-gray-800">We only show one ad, because your site deserves to look great. Rest easy knowing that you fully control how to integrate the ad into your site.</p>
```
Comment (davidfischer) [inline ethicalads-theme/templates/ea/publishers.html:104]: :+1:

File: ethicalads-theme/templates/ea/publishers.html [hunk 5]
```diff
@@ -163,7 +163,7 @@ <h4 class="font-weight-bold" id="faq-policy">Any policies on showing ads?</h4>
       <p>When you show an ad on your site:</p>
       <ul>
         <li>
-          EthicalAds should be the only ad on your site.
+          EthicalAds should be the only ad network on your site (your own promotions are file)
```
Comment (davidfischer) [inline ethicalads-theme/templates/ea/publishers.html:166]:
```suggestion
          EthicalAds should be the only ad network on your site (your own promotions are fine)
```

**Review by davidfischer — APPROVED** [review]

**Review by ericholscher — COMMENTED** [review]
Review from call:

File: ethicalads-theme/templates/ea/publishers.html [hunk 6]
```diff
@@ -24,13 +24,13 @@
       <div class="col-12 col-md-6">
 
         <!-- Heading -->
-        <h1 class="display-3 font-weight-light" data-aos="fade-left">Developers: make money with your content while respecting user privacy</h1>
+        <h1 class="display-3 font-weight-light" data-aos="fade-left">Developers: join the ad network with no tracking</h1>
```
Comment (davidfischer) [inline ethicalads-theme/templates/ea/publishers.html:27]:
```suggestion
        <h1 class="display-3 font-weight-light" data-aos="fade-left">Developers: join the ad network without tracking</h1>
```

This is just a thought

File: ethicalads-theme/templates/ea/publishers.html [hunk 7]
```diff
@@ -81,10 +81,10 @@ <h3>No Cookies</h3>
         </div>
 
         <!-- Heading -->
-        <h3>Publisher Friendly</h3>
+        <h3>Talk to a human</h3>
 
         <!-- Text -->
-        <p class="text-gray-800 mb-0">We started as a publisher, and still run ethical ads on all our sites. We believe in supporting open source, because we're open source devs ourselves.</p>
+        <p class="text-gray-800 mb-0">Our ad network is run by people who understand web publishing and open source. <strong>A real human is always 1 email away.</strong></p>
```
Comment (davidfischer) [inline ethicalads-theme/templates/ea/publishers.html:87]:
```suggestion
        <p class="text-gray-800 mb-0">Our ad network is run by people who understand web publishing and open source. Unlike other networks, <strong>a real human is 1 email away.</strong></p>
```

Don't feel strongly here but just a thought.

File: ethicalads-theme/templates/ea/publishers.html [hunk 8]
```diff
@@ -303,4 +245,67 @@ <h2 class="font-weight-bold">Become a publisher today</h2>
   </div>
 </section>
 
+<!-- FAQ -->
+<!-- ================================================== -->
+<section class="container py-5">
```
Comment (davidfischer) [inline ethicalads-theme/templates/ea/publishers.html:250]:
```suggestion
<section class="container py-5">
  <h2 class="font-weight-bold mb-8">Still have questions?</h2>
```

**Review by davidfischer — APPROVED** [review]
I added a few small possible tweaks but feel free to ignore them. This is a significant improvement! Let's get this in.

**Pull request merged**